test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -153,6 +153,26 @@ def forward(self, x):
             res2 = self.relu2(self.conv4(res1) + res1)
             return res2
 
+    class SingleLinearModule(torch.nn.Module):
+        def __init__(self, use_bias) -> None:
+            super().__init__()
+            self.linear = nn.Linear(4, 4, bias=use_bias)
+
+        def forward(self, x):
+            return self.linear(x)
+
+    class LinearUnaryModule(torch.nn.Module):
+        def __init__(self, use_bias, postop, inplace_postop) -> None:
+            super().__init__()
+            self.linear = nn.Linear(4, 4, bias=use_bias)
+            if postop is nn.Tanh:
```

Comment:
Do we have `nn.Tanh` postop covered by test?

Reply:
Thanks. It's removed.